Twig/Extension/PostExtension.php

```diff
@@ -86,7 +87,14 @@ public function getPermalink($postId)
 
         $permalinkStructure = $this->optionExtension->getOption('permalink_structure', '')->getValue();
 
-        return $this->replacePostArguments($permalinkStructure, $post);
+        $relativeUrl = $this->replacePostArguments($permalinkStructure, $post);
+        if($isAbsolute) {
```

Comment:
Can you please add a whiteline before the `if` statement and a space after the if please?

```php

if ($isAbsolute) {
```